webapp/src/components/MeetingQuickView.tsx

```diff
@@ -0,0 +1,73 @@
+import { Stack } from '@mui/material';
+import React, { useContext, useState } from 'react';
+
+import { EntityId, Meeting as APIMeeting } from '../types/api';
+import CreateMeetingNoteForm from './CreateMeetingNoteForm';
+import SessionContext from './SessionContext';
+import useApiData from '../helpers/useApiData';
+
+interface MeetingQuickViewProps {
+  meetingId?: EntityId;
+}
+
+const MeetingQuickView = ({ meetingId }: MeetingQuickViewProps) => {
+  const { principalId } = useContext(SessionContext);
+  const [changedMeetingNoteIds, setChangedMeetingNoteIds] = useState<
+    EntityId[]
+  >([]);
+  const { data: meeting, error } = useApiData<APIMeeting>({
+    deps: [changedMeetingNoteIds],
+    path: `/meetings/${meetingId}`,
+    sendCredentials: true,
+  });
+  if (error) {
+    return <div>There was an error loading the meeting.</div>;
+  }
+  if (!meeting) {
+    return null;
+  }
+  const currentParticipantId = meeting.participants.find(
+    ({ principal_id }) => principal_id === principalId
+  )?.id;
+  let nextMeetingNoteSortOrder = 1;
+  for (let i = meeting.meeting_notes.length - 1; i >= 0; i--) {
+    const meetingNote = meeting.meeting_notes[i];
+    if (meetingNote.agenda_owning_participant_id === currentParticipantId) {
+      nextMeetingNoteSortOrder = meetingNote.sort_order + 1;
+      break;
+    }
+  }
+  return (
+    <Stack spacing={1}>
+      {meeting.meeting_notes.map((meetingNote, index, meetingNotes) => (
+        <React.Fragment key={meetingNote.id}>
+          {(index === 0 ||
+            meetingNote.agenda_owning_participant_id !==
+              meetingNotes[index - 1].agenda_owning_participant_id) && (
+            <b>
+              {meetingNote.agenda_owning_participant_id === null
+                ? 'Action items'
+                : meetingNote.agenda_owning_participant_id ===
+                  currentParticipantId
+                ? 'Your agenda items'
+                : 'Their agenda items'}
+            </b>
+          )}
+          <ul>
```

Comment:
Is each meeting note list item wrapped in a separate list? Shouldn't items be grouped into the same list? Do they even need to be list items?

Reply:
No, lists would create bullet points at the beginning of each new list item.  

Block-level elements would place each meeting note on it's own separate line.  Each note was changed to be wrapped inside `<div>` tags.  however I believe a `<Box>` MUI component would allow us to include some padding to indent the notes.  What do you think?

Reply:
There is a roadmap item to allow each note to be indented at a different level. The `<Box>` component would be a good fit for this. Sounds like a good choice.

Reply:
Thanks.